packages/web/src/common/design/components/Switch.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import { styled } from "../../../modules/theming/themes"
+import { Theme } from "../../../modules/theming/types/Theme"
+
+const size = 30
+
+interface checkboxProps {
```

Comment:
interfaces should start with a capital char
```suggestion
interface CheckboxProps {
```